packages/ui/package.json

```
@@ -0,0 +1,51 @@
+{
+  "name": "@dzcode.io/ui",
+  "description": "dzcode ui library",
+  "version": "1.0.0",
+  "author": {
+    "email": "sohaib.code@gmail.com",
+    "name": "Souhaib Benbouzid",
+    "url": "https://sohaibbenbouzid.com/"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.7.0",
+    "@emotion/styled": "^11.6.0",
+    "@fontsource/roboto": "^4.5.1",
+    "@mui/icons-material": "^5.2.0",
+    "@mui/material": "^5.2.2",
+    "@types/uuid": "^8.3.3",
+    "react": "^17.0.2",
+    "stylis-plugin-rtl": "^1"
+  },
+  "devDependencies": {
+    "@dzcode.io/tooling": "*",
```

you're missing `@dzcode.io/models` here
```suggestion
    "@dzcode.io/models": "*",
    "@dzcode.io/tooling": "*",
```

---

d8969231f722091d8648dc39c9a2e424a5429a64